src/http-spec.ts

```diff
@@ -39,11 +39,16 @@ export interface IHttpOperation extends INode {
   deprecated?: boolean;
 }
 
+export type ParametersWithSchema<T> = {
+  values: T[];
+  schema: JSONSchema7;
+};
+
 export interface IHttpOperationRequest {
-  path: IHttpPathParam[];
-  query: IHttpQueryParam[];
-  headers: IHttpHeaderParam[];
-  cookie: IHttpCookieParam[];
+  path: ParametersWithSchema<IHttpPathParam>;
```

Comment:
I know you hate the I prefix, but should this have one? Personally I try and stuck to consistency over preference, which is of course a preference, but... 😋

Reply:
Hmm, what prefix?

Reply:
Aaah you mean `I` prefix, alright no problem